lib/sandboxed_renderer/init.ts

```diff
@@ -89,6 +89,9 @@ Object.defineProperty(preloadProcess, 'noDeprecation', {
   }
 });
 
+const contextId = v8Util.getHiddenValue<string>(global, 'contextId');
+Object.defineProperty(process, 'contextId', { get: () => contextId });
```

Comment:
does this need to be defined on `preloadProcess`?

Reply:
ah nevermind, i see this is handled below.

Reply:
yes, this is exactly why whole change is necessary as the binding is only available on the internal process object, not to the preload script.